src/wallet/rpcwallet.cpp

```diff
@@ -2540,15 +2554,57 @@ static UniValue loadwallet(const JSONRPCRequest& request)
     return obj;
 }
 
-static UniValue createwallet(const JSONRPCRequest& request)
+static UniValue setwalletflag(const JSONRPCRequest& request)
 {
+    std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
+    CWallet* const pwallet = wallet.get();
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
+        std::string flags = "";
+        for (auto& it : walletflagmap) flags += (flags == "" ? "" : ", ") + it.first;
+        throw std::runtime_error(
+            "setwalletflag flag ( state )\n"
+            "\nChange the state of the given wallet flag for a wallet.\n"
+            "\nArguments:\n"
+            "1. flag    (string, required) The name of the flag to change. Current available flags: " + flags + "\n"
+            "2. value   (boolean, optional, default=true) The new state.\n"
+            "\nResult:\n"
+            "true|false (boolean) True or false, reflecting whether the flag was modified (true), or (false) whether it was already set to the given state\n"
```

Comment:
I think being told that your action did nothing vs your action did something is a good UI, personally.

Perhaps it should actually *throw* instead of returning false, though...